components/Comments/components/Comment/Comment.tsx

```diff
@@ -1,19 +1,26 @@
 import { Avatar } from 'components';
 import { Divider } from 'components/Divider/Divider';
+import { useRouter } from 'next/router';
 import { FC } from 'react';
 import { toDateString } from 'utils';
 
 import classes from './Comment.module.scss';
 import { TCommentProps } from './Comment.types';
 
 export const Comment: FC<TCommentProps> = ({ author, date, text }) => {
+  const router = useRouter();
+  const clickHandler = (): void => {
+    router.push(`/blogs/${author.id}`);
+  };
   return (
     <>
       <div className={classes.comment}>
         <div className={classes.author}>
-          <Avatar imgUrl='' size='small' />
+          <Avatar imgUrl={author.imgUrl} size='small' />
           <div className='ms-1'>
-            <h5 className='m-0'>{author.name}</h5>
+            <h5 onClick={clickHandler} className='m-0 link pointer'>
```

Comment:
wrap it with Link component, do not add click handler. See other places where Link component is used